core/trino-server/src/main/provisio/trino.xml

```diff
@@ -68,6 +68,12 @@
         </artifact>
     </artifactSet>
 
+    <artifactSet to="plugin/example-jdbc">
```

Comment:
I don't think it should be part of the product

Reply:
I only added this to be consistent with how we include the other example plugin: https://github.com/trinodb/trino/blob/master/core/trino-server/src/main/provisio/trino.xml#L65

I'm ok with removing this, but I expected it to be brought up during review. Please confirm and I'll drop it.

Reply:
Please remove it from product.